### PR TITLE
i18n(ja): Update components.mdx

### PR DESCRIPTION
**Pull request by morinokami**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- Update components.mdx
  - https://github.com/withastro/starlight/pull/397

### CODE REVIEW

File: docs/src/content/docs/ja/guides/components.mdx [hunk 1]
```diff
@@ -116,6 +116,50 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 
 :::
 
+### リンクカード
+
+`<LinkCard>`を使用すると、他のページへのリンクを目立たせることができます。
+
+`<LinkCard>`は`title`と[`href`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#href)属性を必須とします。短い`description`や、`target`など他のリンク属性も指定できます。
```
Comment (delucis) [inline docs/src/content/docs/ja/guides/components.mdx:123]: I think we could link to the same MDN docs but in the Japanese translation here:

```suggestion
`<LinkCard>`は`title`と[`href`](https://developer.mozilla.org/ja/docs/Web/HTML/Element/a#href)属性を必須とします。短い`description`や、`target`など他のリンク属性も指定できます。
```

**Review by delucis — APPROVED** [review]
Thank you @morinokami — this looks good! ✨ 

Just noticed one opportunity for a small improvement.

**Comment by morinokami** [comment]
@delucis Thanks for spotting this! Merged your suggestion.

**Review by delucis — APPROVED** [review]
Perfect!

**Pull request merged**